workers/unity/Assets/Playground/Improbable.Playground.asmdef

```diff
@@ -9,7 +9,8 @@
         "Unity.Entities",
         "Unity.Entities.Hybrid",
         "Improbable.CSharp",
-        "Unity.Mathematics"
+        "Unity.Mathematics",
+        "Improbable.Gdk.Android"
```

Comment:
move it up to the other `Improbable.Gdk....` references to keep some kind of ordering